plugins/item_previews/web_client/js/item_previews.js

```diff
@@ -9,6 +9,15 @@ girder.views.ItemPreviewWidget = girder.View.extend({
         'click a.g-item-preview-link': function (event) {
             var id = this.$(event.currentTarget).data('id');
             this.parentView.itemListView.trigger('g:itemClicked', this.collection.get(id));
+        },
+        'click .g-widget-item-previews-wrap': function (event) {
+            this.wrapPreviews = !this.wrapPreviews;
+            this.render();
+            if (this.wrapPreviews) {
+                this.$('.g-widget-item-previews-container')[0].style.whiteSpace = 'normal';
+            } else {
+                this.$('.g-widget-item-previews-container')[0].style.whiteSpace = 'nowrap';
+            }
```

Comment:
Let's use `addClass` and `removeClass` to separate style rules from the view logic.